From 084d8ae2f349a7621e1e51c81b4494b123f3c09a Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Fri, 25 Aug 2017 14:58:35 +0800
Subject: [PATCH 2/2] fs: yaffs2: includes the missing header file

To fix the following build error:
  fs/yaffs2/yaffs_vfs.c: In function 'yaffs_mknod':
  fs/yaffs2/yaffs_vfs.c:1252:2: error: implicit declaration of function 'current_fsuid' [-Werror=implicit-function-declaration]
    uid_t uid = YCRED_FSUID();
    ^

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 fs/yaffs2/yaffs_vfs.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fs/yaffs2/yaffs_vfs.c b/fs/yaffs2/yaffs_vfs.c
index 8b2019fe3c2c..c7be4cddd1de 100644
--- a/fs/yaffs2/yaffs_vfs.c
+++ b/fs/yaffs2/yaffs_vfs.c
@@ -167,6 +167,7 @@ static uint32_t YCALCBLOCKS(uint64_t partition_size, uint32_t block_size)
 
 #include <linux/uaccess.h>
 #include <linux/mtd/mtd.h>
+#include <linux/cred.h>
 
 #include "yportenv.h"
 #include "yaffs_trace.h"
-- 
2.5.0

